components/scheduler/src/Scheduler.svelte

```diff
@@ -68,6 +70,19 @@
     notification_subject: "Invitation",
     recurrence: "none",
     recurrence_cadence: ["none"],
+    custom_fields: [
+      {
+        title: "Your Name",
+        type: "text",
+        required: false,
+      },
+      {
+        title: "Email Address",
+        type: "text",
+        required: true,
+        placeholder: "you@example.com",
```

Comment:
It looks like the `CustomField` type is missing `placeholder` attribute?

Reply:
Good catch!